_topic_map.yml

```diff
@@ -1043,6 +1043,9 @@ Topics:
 ### Autoscaling
 #- Name: Configuring Knative Serving autoscaling
 #  File: configuring-knative-serving-autoscaling
+### Knative Client
+#- Name: Knative Client
```

Comment:
I'd still suggest to change this from "Knative Client" to "Using Knative Client" to be consistent with the assembly names throughout our doc set.